src/mscorlib/ref/mscorlib.csproj

```diff
@@ -23,7 +23,6 @@
     
     <ErrorReport>prompt</ErrorReport>
     <Optimize Condition="'$(Optimize)' == ''">true</Optimize>
-    <GenerateNativeVersionInfo Condition="'$(OsEnvironment)'=='Windows_NT'">true</GenerateNativeVersionInfo>
```

Comment:
This shouldn't affect native binaries at all, given that this will only affect mscorlib managed build. For native assets, we call this target when we call build.cmd/sh script before the native build to ensure that the assets are there on build time.